lib/futurism/resolver/resources.rb

```diff
@@ -21,14 +21,24 @@ def resolve
 
         @resources_without_sgids.each do |resource_definition|
           resource = lookup_resource(resource_definition)
-          html = renderer_for(resource_definition: resource_definition).render(resource)
+          renderer = renderer_for(resource_definition: resource_definition)
+          html =
+            begin
+              renderer.render(resource)
+            rescue => exception
+              error_render.render(exception)
+            end
```

Comment:
Note: I thought about rendering fewer exceptions, perhaps limiting it to only `ActionView` related errors. But syntax errors or stack overflows and other errors would then not render in the UI and only be present in the logs.